atlasdb-timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServer.java

```diff
@@ -56,6 +58,12 @@ public static void main(String[] args) throws Exception {
     }
 
     @Override
+    public void initialize(Bootstrap<TimeLockServerConfiguration> bootstrap) {
+        super.initialize(bootstrap);
```

Comment:
No-one seems to call `super.initialize` in other repos as the `Application.initialize` is known to be empty.